app/controllers/api/v1/trucking/inventory_items_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Api
+  module V1
+    module Trucking
```

Comment:
So to be clear, the directory would then be: 
app/controllers/api/v1/inventory/items_controller.rb